1/README.md

```diff
@@ -89,6 +89,8 @@ C4 is meant to provide a reusable optimal collaboration model for open source so
 
 ### 2.6. Evolution of Public Contracts
 
+Definition: **Public Conracts** _noun_ - All public APIs and protocols. In other words, read the BEPs (and the things they reference) and you'll have a good start. In particular, see BEP-7 - Definition of the BigchainDB Public API.
```

Comment:
Great! Should it belong to `2.1`? All definitions are there. (PS: typo "Conracts").

Reply:
Oh interesting, I didn't notice that some other things were defined there (like "Contributor" and "Maintainer"). Anyway, I think it's fine to put this definition here, right when it's needed, in context.

Reply:
Oops Contracts!